src/index.js

```diff
@@ -0,0 +1,165 @@
+  } else if (rangeVal.value >= 70) {
+    document.getElementById('city-circle').style.background = '#E8BCC3';
+    document.getElementById('temp-circle').style.background = '#E8BCC3';
+    document.getElementById('sky-circle').style.background = '#E8BCC3';
+    document.getElementById('weather').style.color = '#E8BCC3';
+  } else if (rangeVal.value >= 60) {
+    document.getElementById('city-circle').style.background = '#E5DDE7';
+    document.getElementById('temp-circle').style.background = '#E5DDE7';
+    document.getElementById('sky-circle').style.background = '#E5DDE7';
+    document.getElementById('weather').style.color = '#E5DDE7';
+  } else if (rangeVal.value >= 50) {
+    document.getElementById('city-circle').style.background = '#CECEE0';
+    document.getElementById('temp-circle').style.background = '#CECEE0';
+    document.getElementById('sky-circle').style.background = '#CECEE0';
+    document.getElementById('weather').style.color = '#CECEE0';
+  } else if (rangeVal.value < 50) {
+    document.getElementById('city-circle').style.background = '#AEBFDD';
+    document.getElementById('temp-circle').style.background = '#AEBFDD';
+    document.getElementById('sky-circle').style.background = '#AEBFDD';
+  }
+};
+
+const changeRangeS = () => {
+  const input = document.getElementById('range');
+  input.addEventListener('input', (event) => {
+    if (input.value >= 80) {
+      document.getElementById('city-circle').style.background = '#EA9A9F';
+      document.getElementById('temp-circle').style.background = '#EA9A9F';
+      document.getElementById('sky-circle').style.background = '#EA9A9F';
+      document.getElementById('weather').style.color = '#EA9A9F';
+    } else if (input.value >= 70) {
+      document.getElementById('city-circle').style.background = '#E8BCC3';
+      document.getElementById('temp-circle').style.background = '#E8BCC3';
+      document.getElementById('sky-circle').style.background = '#E8BCC3';
+      document.getElementById('weather').style.color = '#E8BCC3';
+    } else if (input.value >= 60) {
+      document.getElementById('city-circle').style.background = '#E5DDE7';
+      document.getElementById('temp-circle').style.background = '#E5DDE7';
+      document.getElementById('sky-circle').style.background = '#E5DDE7';
+      document.getElementById('weather').style.color = '#E5DDE7';
+    } else if (input.value >= 50) {
+      document.getElementById('city-circle').style.background = '#CECEE0';
+      document.getElementById('temp-circle').style.background = '#CECEE0';
+      document.getElementById('sky-circle').style.background = '#CECEE0';
+      document.getElementById('weather').style.color = '#CECEE0';
+    } else if (input.value < 50) {
+      document.getElementById('city-circle').style.background = '#AEBFDD';
+      document.getElementById('temp-circle').style.background = '#AEBFDD';
+      document.getElementById('sky-circle').style.background = '#AEBFDD';
+    }
+  });
+};
```

Comment:
There's a lot of repetition in this function. Notice how in each conditional, you are setting the same three elements to the same background color? We can DRY up our code by using the conditionals to instead change the value of a variable and THEN set that color to the elements backgrounds.

```Javascript
const changeRanges = () => {
  const input = document.getElementById('range');
  input.addEventListener('input', (event) => {
    let color = ''
    if (input.value >= 80) {
      color = '#EA9A9F';
    } else if (input.value >= 70) {
      color = '#E8BCC3';
    } else if (input.value >= 60) {
      color = '#E5DDE7';
    } else if (input.value >= 50) {
      color = '#CECEE0';
    } else if (input.value < 50) {
      color = '#AEBFDD';
    }
    document.getElementById('city-circle').style.background = color;
    document.getElementById('temp-circle').style.background = color;
    document.getElementById('sky-circle').style.background = color;
  });
};
```